website/content/en/preview/tasks/amitasks.md

```diff
@@ -97,7 +116,7 @@ Keep in mind, that this could prevent you from getting critical security patches
 
 ### Task 3: Control the pace of node disruptions
 
-To help prevent the possibilities of a new AMI being deployed to all your nodes and breaking all of your workloads, you can enable Karpenter [**Disruption Budgets**]({{< relref "../concepts/disruption/#disruption-budgets " >}}).
+To reduce the risk of a new AMI being deployed immediately to all your nodes and breaking all of your workloads, you can enable Karpenter [**Disruption Budgets**]({{< relref "../concepts/disruption/#disruption-budgets " >}}).
```

Comment:
I know this is a suggested change I gave in offline review, but after rereading it I realize I was applying the risk to breaking workloads rather than rolling out new AMIs immediately across all nodes. Thoughts on something like this?
```suggestion
To reduce the risk of entire workloads being degraded when a new AMI is deployed, you can enable Karpenter [**Disruption Budgets**]({{< relref "../concepts/disruption/#disruption-budgets " >}}).
```

Reply:
Done.